packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts

```diff
@@ -0,0 +1,61 @@
+import { createStyles, keyframes } from '$lib/styles';
+import type { SvelteUINumberSize, SvelteUIColor, ColorShades } from '$lib/styles';
+
+export interface BreadcrumbItemStylesParams {
+	color: SvelteUIColor;
+	size: SvelteUINumberSize;
+}
+
+
+export const sizes = {
+	xs: 3,
+	sm: 5,
+	md: 8,
+	lg: 12,
+	xl: 16
+};
+
+export const fontSizes = {
+	xs: 12,
+	sm: 14,
+	md: 16,
+	lg: 18,
+	xl: 20
+};
+
+export default createStyles(
+	(theme, { color, size }: BreadcrumbItemStylesParams) => ({
+		root: {
+			position: 'relative',
+			'& .wrapper': {
```

Comment:
instead of like this, you do like:
`root: {...},
 wrapper: {...},
 innerText: {...}
` 
and then instead of `cx('wrapper'...)`  you do `cx(classes.wrapper,...)`. This generates a different class in the DOM, which is the behaviour we want